turbo.json

```
@@ -59,6 +59,8 @@
     },
     "dev": {
       "cache": false
+    },
+    "start": {
```

I think we should disable Turborepo caching just like in `dev` command:

```suggestion
    "start": {
      "cache": true
```